aws_eden_cli/dynamodb.py

```diff
@@ -0,0 +1,102 @@
+import logging
```

Comment:
If possible, could you move below lines to this file as well?

https://github.com/baikonur-oss/aws-eden-cli/blob/6299497b2f90ecbb2787a6f3aee474d3bf64b0f2/aws_eden_cli/cmdline.py#L137-L150
https://github.com/baikonur-oss/aws-eden-cli/blob/6299497b2f90ecbb2787a6f3aee474d3bf64b0f2/aws_eden_cli/cmdline.py#L161-L173